BTPowershell.psm1

```diff
@@ -0,0 +1,55 @@
+﻿# Set strict mode.
+Set-StrictMode -Version 2.0
+
+# Disable progress bar.
+$ProgressPreference = "SilentlyContinue"
+
+# Set default ErrorAction to Stop.
+$ErrorActionPreference = "Stop"
+
+################################################################################
+# Get all customer end users based on the name of the customer
+################################################################################
+function Get-EndUsers($ticket, $customerCompanyName)
+{
+    try
+    {
+        DisplayInProgressOperation "Get end users"
+
+        # Get the list of end users.
+        $endUsers = Get-BT_CustomerEndUser -Ticket $ticket -CustomerCompanyName $customerCompanyName -RetrieveAll
+
+        DisplayCompletedOperation "OK" Green
+        return $endUsers
+    }
+    catch
+    {
+        DisplayCompletedOperation "FAIL" Red
+        Write-Host -ForegroundColor Red "FATAL: Export-CustomerEndUserListToCsv failed with $_"
+        return $false
+    }
+}
+
+################################################################################
+# Export List of Customer End Users to CSV
+################################################################################
+function Export-CustomerEndUsersToCsv($endUsers, $csvPath = "${env:USERPROFILE}\Desktop\CustomerCSV.csv")

```

Comment:
This function does not belong to the BTPowerShell. This is an utility function that takes a list of *something* and write it to a csv file. It could be made way more generic and added to a generic data manipulation module.